app/models/account.rb

```diff
@@ -501,6 +501,19 @@ def ensure_keys!
     save!
   end
 
+  def private_key
```

Comment:
I assume this is here because all of:

- We dont want to change every single access spot to go through the new association
- We might have a scenario where some accounts have connected `account_secret` records and others do, hence the conditional and super access

Correct?

If so, would the plan here be to ultimately migrate and either `delegate` this, or otherwise clean up the access?

Reply:
It's here for the transition period when some accounts don't have an associated account secret yet.